src/i18n/resources/en/fra.js

```diff
@@ -7,6 +7,16 @@ module.exports = {
   forestArea100HaYear: 'Forest area (1000 ha)',
 
   // tables
+  areaAffectedByFire: {
+    areaUnitLabel2025: 'Area affected (1000 ha)',
+  },
+
+  disturbances: {
+    disturbances2025: 'Forest damage',
```

Comment:
can this be forestDamage ?

Reply:
Definitely, at first I named it 'forestDamage' but renamed it to keep track of what has changed from previous version

I will change this